From ben@fluff.org.uk Tue Mar 21 14:56:19 2006
Date: Tue, 21 Mar 2006 22:54:47 +0000
From: Ben Dooks <ben-linux-usb@fluff.org>
To: linux-usb-devel@lists.sourceforge.net, greg@kroah.com
Subject: USB: cleanups for ohci-s3c2410.c
Message-ID: <20060321225447.GA6198@home.fluff.org>
Content-Disposition: inline


Fix compile errors due to functions not being
defined static

Signed-off-by: Ben Dooks <ben-linux@fluff.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-s3c2410.c |    9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

--- gregkh-2.6.orig/drivers/usb/host/ohci-s3c2410.c
+++ gregkh-2.6/drivers/usb/host/ohci-s3c2410.c
@@ -37,7 +37,7 @@ static void s3c2410_hcd_oc(struct s3c241
 
 /* conversion functions */
 
-struct s3c2410_hcd_info *to_s3c2410_info(struct usb_hcd *hcd)
+static struct s3c2410_hcd_info *to_s3c2410_info(struct usb_hcd *hcd)
 {
 	return hcd->self.controller->platform_data;
 }
@@ -316,7 +316,8 @@ static void s3c2410_hcd_oc(struct s3c241
  *
 */
 
-void usb_hcd_s3c2410_remove (struct usb_hcd *hcd, struct platform_device *dev)
+static void
+usb_hcd_s3c2410_remove (struct usb_hcd *hcd, struct platform_device *dev)
 {
 	usb_remove_hcd(hcd);
 	s3c2410_stop_hc(dev);
@@ -334,8 +335,8 @@ void usb_hcd_s3c2410_remove (struct usb_
  * through the hotplug entry's driver_data.
  *
  */
-int usb_hcd_s3c2410_probe (const struct hc_driver *driver,
-			   struct platform_device *dev)
+static int usb_hcd_s3c2410_probe (const struct hc_driver *driver,
+				  struct platform_device *dev)
 {
 	struct usb_hcd *hcd = NULL;
 	int retval;
